docs/data-sources/resources.md

```diff
@@ -15,6 +15,11 @@ Resources in Twingate represent servers on the private network that clients can
 ```terraform
 data "twingate_resources" "foo" {
   name = "<your resource's name>"
+#  name_regexp = "<your resource's name>"
```

Comment:
Update with
#  name_regexp = "<regular expression of resource name>"
#  name_contains = "<a string in the resource name>"
#  name_exclude = "<your resource's name to exclude>"
#  name_prefix = "<prefix of resource name>"
#  name_suffix = "<suffix of resource name>"